File: app/src/main/java/com/example/adoteme_app/auth/presentation/login_screen/auth_service/Request/LoginRequest.java
package com.example.adoteme_app.auth.presentation.login_screen.auth_service.Request;

public class LoginRequest {
    private String email;
    private String senha;

    public LoginRequest() {}

    public LoginRequest(String email, String senha) {
        this.email = email;
        this.senha = senha;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }
}

File: app/src/main/java/com/example/adoteme_app/auth/presentation/login_screen/auth_service/request/LoginRequest.kt
package com.example.adoteme_app.auth.presentation.login_screen.auth_service.Request

class LoginRequest {
    var email: String? = null
    var senha: String? = null

    constructor()

    constructor(email: String?, senha: String?) {
        this.email = email
        this.senha = senha
    }
}